examples/my_kentik_portal_example_test.go

```diff
@@ -23,26 +23,27 @@ func runCRUDExample() error {
 	if err != nil {
 		return err
 	}
-	var tenant_id models.ID = 577
+	// TODO(mpalczynski): pick tenantID from a list of all tenants
```

Comment:
Maybe it's worth to create JIRA ticket for that? It would be nice for examples to work on other environments

Reply:
https://clkentik.atlassian.net/browse/KNTK-373